src/views/tickets/email.php

```diff
@@ -369,7 +374,7 @@
 																		<span style="color:#0a0a0e !important; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px; display:block; margin-bottom:5px;"><?php echo $venue_name; ?></span>
 																		<a style="color:#006caa !important; display:block; margin:0; font-family: 'Helvetica Neue', Helvetica, sans-serif; font-size:13px; text-decoration:underline;">
 																			<?php echo $venue_address; ?><br />
-																			<?php echo $venue_city; ?>
+																			<?php printf( '%s, %s %s', $venue_city, $venue_state, $venue_zip ); ?>
```

Comment:
If both the state and the zip are missing, the following would be printed:

```
Phoenix,
```

We might need to do something like:

```php
<?php
if ( $venue_city && ( $venue_state || $venue_zip ) ) :
  printf( '%s, %s %s', $venue_city, $venue_state, $venue_zip );
else:
  echo $venue_city;
endif;
```